### DISCUSSION TITLE
[@types/dedent] Currently Doesn't Work in ESModules

### DISCUSSION THREAD

**Discussion by neurolag** [DefinitelyTyped/DefinitelyTyped#60592] [Issues with a @types package] [answered]
Currently, the `@types/dedent` declarations won't work in esmodules.
As far as I can see, it's caused by `@types/dedent`s `package.json` file not including a `types` property.

Adding it (`"types": "index.d.ts"`) seems to fix this issue.

[a comment by an automated account is not shown]

**Comment by neurolag** [comment, marked as answer]
@andrewbranch, @basarat, @blakeembrey, @borisyankov, @DanielRosenwasser, @Diullei, @horiuchi, @Jessidhia, @johnnyreilly, @kieferrm, @peterblazejewicz, @RyanCavanaugh, @sebastian-lenz, @staxmanade, @Steve-Fenton, @vvakame

I'm terribly sorry for pinging you guys - I couldn't find a list of DT maintainers so I decided to ping all members of the organization.

I think this is more likely an issue with the `@types/dedent` being very old (published 5 years ago) rather than the type declaration itself.
Is there some way for you to publish a new patch version of `@types/dedent` manually?

**Reply by andrewbranch** [reply]
This is a TypeScript bug; it will be fixed in a 4.7.3 patch release soon. https://github.com/microsoft/TypeScript/issues/49326

**Reply by peterblazejewicz** [reply]
I've made anyway changes to the module to support global script usage (I works in the browser via script):
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60604/files

**Reply by neurolag** [reply]
> This is a TypeScript bug; it will be fixed in a 4.7.3 patch release soon. [microsoft/TypeScript#49326](https://github.com/microsoft/TypeScript/issues/49326)

Awesome! Thanks a lot for letting me know 😄
